astropy/visualization/interval.py

```diff
@@ -6,39 +6,73 @@
 """
 
 from __future__ import division, print_function
-
```

Comment:
I'm not a fan of the blank lines within the imports.  I wasn't aware that was recommended.